doc/source/cluster/cluster_under_construction/getting-started.rst

```diff
@@ -42,56 +35,39 @@ Where to go from here?
     :column: col-lg-6 px-2 py-2
     :card:
 
-    **Quick Start** 
+    **I want to run Ray on a cloud provider** 
     ^^^
-
-    In this quick start tutorial you will take a sample application designed to
-    run on a laptop and scale it up in the cloud.
+    This guide helps you take a sample application designed to run on a laptop and
+    scale it up in the cloud. Access to an AWS or GCP account is required.
 
     +++
     .. link-button:: ref-cluster-quick-start-vms-under-construction
         :type: ref
-        :text: Ray Clusters Quick Start
+        :text: Getting Started with Ray Clusters on VMs
         :classes: btn-outline-info btn-block
     ---
 
-    **Key Concepts**
+    **I want to run Ray on Kubernetes**
     ^^^
-
-    Understand the key concepts behind Ray Clusters. Learn about the main
-    concepts and the different ways to interact with a cluster.
+    This guide helps you deploy a Ray application to a Kubernetes cluster.
+    You can run the tutorial on a remote Kubernetes cluster or on your laptop via KinD.
 
     +++
-    .. link-button:: cluster-key-concepts
+    .. link-button:: kuberay-quickstart
         :type: ref
-        :text: Learn Key Concepts
+        :text: Getting Started with Ray on Kubernetes
         :classes: btn-outline-info btn-block
     ---
 
-    **Deployment Guide**
+    **I want to learn key Ray concepts**
     ^^^
-
-    Learn how to set up a distributed Ray cluster and run your workloads on it.
-
-    +++
-    .. link-button:: ref-deployment-guide
-        :type: ref
-        :text: Deploy on a Ray Cluster
-        :classes: btn-outline-info btn-block
-    ---
-
-    **API**
-    ^^^
-
-    Get more in-depth information about the various APIs to interact with Ray
-    Clusters, including the :ref:`Ray cluster config YAML and CLI<cluster-config>`,
-    the :ref:`Ray Client API<ray-client>` and the
-    :ref:`Ray job submission API<ray-job-submission-api-ref>`.
+    Understand the key concepts behind Ray Clusters. Learn about the main
+    concepts and the different ways to interact with a cluster.
```

Comment:
This description could be made less redundant -- for example, the word "concepts" should appear just once.

Reply:
(Covered by Richard's suggestion below.)